get_soccer_players_articles.py

```diff
@@ -9,26 +9,44 @@
 
 DEBUG = True
 
-
+''' Returns a list with wiki ids from the input file filein'''
 def load_wiki_ids(filein):
     with open(filein) as i:
         return [l.strip() for l in i.readlines()]
 
-
+'''Prints out article present in path corpus directory if its ID is present in "soccer_ids" list'''
 def extract_soccer_articles(soccer_ids, corpus_dir, output_dir):
+    # Iterating over the files and sub-directories from corpus directory using os.walk function
     for path, subdirs, files in os.walk(corpus_dir):
+        # Iterating over all the files
         for name in files:
             f = os.path.join(path, name)
-            with open(f) as i:
+            # Opening a single file f
+            with open(f,'r') as i:
                 content = ''.join(i.readlines())
-            match = re.search('id="([^"]+)"', content)
-            current_id = match.group(1)
-            if DEBUG:
-                print "File = [%s] - Wiki ID = [%s]" % (f, current_id)
-            if current_id in soccer_ids:
-                shutil.copy(f, output_dir)
+            # Finding ALL the doc ids in content using a simple regex.
```

Comment:
Yes, if you read the open tasks instructions, we would not be discussing 
about this now.

On 3/17/15 2:46 PM, Enayat Ullah wrote:
> In get_soccer_players_articles.py
> <https://github.com/dbpedia/fact-extractor/pull/16#discussion_r26572897>:
>
>>                  content = ''.join(i.readlines())
>> -            match = re.search('id="([^"]+)"', content)
>> -            current_id = match.group(1)
>> -            if DEBUG:
>> -                print "File = [%s] - Wiki ID = [%s]" % (f, current_id)
>> -            if current_id in soccer_ids:
>> -                shutil.copy(f, output_dir)
>> +            # Finding ALL the doc ids in content using a simple regex.
>
> Okay! Yes I did! Thanks for the clarification! So, this issue just
> involves the addition of comments and docstrings with no major
> modification in code, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dbpedia/fact-extractor/pull/16/files#r26572897>.
>